oo-install/site_assets/oo-install-bootstrap.sh

```diff
@@ -61,13 +61,18 @@ elif [[ $RUBYVER == ruby\ 2\.* ]]
 then
   RUBYDIR=''
 fi
-GEM_PATH="${TMPDIR}INSTALLPKGNAME/vendor/bundle/ruby/${RUBYDIR}gems/"
+
+OOINSTALL_GEM_PATH="${TMPDIR}INSTALLPKGNAME/vendor/bundle/ruby/${RUBYDIR}gems/"
 RUBYLIB="${TMPDIR}INSTALLPKGNAME/lib:${TMPDIR}oo-install/vendor/bundle"
-for i in `ls $GEM_PATH`
+for i in `ls $OOINSTALL_GEM_PATH`
 do
-  RUBYLIB="${RUBYLIB}:${GEM_PATH}${i}/lib/"
+  RUBYLIB="${RUBYLIB}:${OOINSTALL_GEM_PATH%/}/${i}/lib/"
```

Comment:
@tiwillia I don't understand why we set the RUBYLIB for the libs in OOINSTALL_GEM_PATH.  Was the layout of the directory not what rubygems is expecting?  

Do we need to worry about the same thing for the libs found in each of the paths in the original GEM_PATH?